public/app/core/components/AppChrome/MegaMenu/MegaMenuItemText.tsx

```diff
@@ -38,6 +38,7 @@ export function MegaMenuItemText({ children, isActive, onClick, target, url }: P
       href={url}
       target={target}
       onClick={onClick}
+      {...(isActive && { 'aria-current': 'page' })}
```

Comment:
This might be _slightly_ inaccurate, as the href of the active item doesn't exactly match the current page (e.g. it's selecting a link that refers to `/dashboards`, but we might be on `/d/someuid/name`). 
Does this seem right? Or just `aria-current=true`?